docker-image/Dockerfile.arm64

```diff
@@ -13,15 +13,12 @@ COPY --from=qemu /usr/bin/qemu-aarch64-static /usr/bin/
 
 # Since our binary isn't designed to run as PID 1, run it via the tini init daemon.
 ENV TINI_VERSION v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /sbin/tini
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static-arm64 /sbin/tini
 RUN chmod +x /sbin/tini
 
-# Ensure musl C library is installed 
-RUN apt-get update && apt-get install musl && ln -s /lib/ld-musl-aarch64.so.1 /lib/libc.musl-aarch64.so.1
-
 FROM scratch
+COPY --from=base /tmp /tmp
```

Comment:
did you figure out what `/tmp` is for?

Reply:
Yeah this was a bug fix: 
https://github.com/projectcalico/typha/pull/211

Needless to say that `/tmp` must exist. However depending on this whole `base` stuff if there's no other reason for `base` I can just create the `/tmp` folder within `scratch`.

Reply:
Going to make this thread as resolved.

Reply:
@stevegaossou would be good to add this /tmp to kube-controllers as well :D

Reply:
Yeah, will do. 

I took a quick look around for best technique for `from scratch` images and it looks like to create a directory with the right permissions what we have here with `from ... base` --> `COPY --from=base /tmp /tmp` is the right way to go. So I'll just copy the same pattern over to kube controllers.

Reply:
I already addressed this with the other PR, so we're good.

Reply:
Ahh, got it. Didn't check the timestamp on your comment and realize it was from before our chat about kube controllers.